smoke_tests/delegations_test.js

```diff
@@ -0,0 +1,226 @@
+// k6 smoke test of api delegations actions
+
+import http from 'k6/http';
+import { check, group, sleep, fail } from 'k6';
+
+export let options = {
+  thresholds: {
+    failedTestCases: [{ threshold: 'count==0' }], // add "abortOnFail: true" to exit immediately
+    'checks': ['rate>0.99'] // the rate of successful checks should be higher than 99%
+  },
+  iterations: 1
+};
+
+const BASE_URL = 'https://api.liquidvoting.io';
+const AUTH_KEY = 'bc7eeccb-5e10-4004-8bfb-7fc68536bbd7';
```

Comment:
this should be a github secret, I'll set it up. Of course, it'll have to be passed into the script somehow. part of what we'll need to discuss on slack

Reply:
${{ secrets.TEST_API_AUTH_KEY }} now available to the github actions